python/paddle/distributed/auto_parallel/context.py

```diff
@@ -429,3 +429,77 @@ def _get_data_parallel_info(self):
     def _get_model_parallel_info(self):
         # This function is a hard code, and will be obsoleted in the future
         return self._model_parallel_axis, self._process_mesh
+
+
+class DistOpContext:
```

Comment:
Please consider to merge the function of DistOpContext into the DistributedContext. At the beginning, we can keep our classes as less as possible. In the future, we can split a large class into several classes to represent different abstract concepts.